### PR TITLE
Documentation for custom image attributes (Bolt PR 7743)

### PR DESCRIPTION
**Pull request by ankedsgn**
Adds documentation for the custom attributes for imagefields.
Goes with: https://github.com/bolt/bolt/pull/7743

### CODE REVIEW

File: docs/fields/image.md [hunk 1]
```diff
@@ -35,21 +35,21 @@ field.
 
 * `extensions` Allows you to restrict users to only be able to upload files with
   certain file extensions.
-* `attrib` Can be set to either of the following:
+* `attrib` Can be set to a custom attribute, such as alt-texts, captions or data-attributes. For example:
   * `alt` Show a text field for the `alt` parameter
-  * `title` Show a text field for the `title` parameter
```
Comment (JarJak) [inline docs/fields/image.md]: title is still a special attribute so I would leave it here

**Review by JarJak — CHANGES_REQUESTED** [review]

**Comment by ankedsgn** [comment]
Title is discouraged to be used due to accessibility reasons, so I opted to go for the more general 'caption' instead, thus showing a practical example.

**Comment by JarJak** [comment]
Let's leave title (since it is special in Bolt's code) together with alt and caption. You can also add a note that title is discouraged if you want.

**Comment by ankedsgn** [comment]
> Let's leave title (since it is special in Bolt's code) together with alt and caption. You can also add a note that title is discouraged if you want.

I agree, It's handled specifically in Bolt's code so it kinda deserves a mention. I put this back in the docs. Adding a note on a11y might be a bit much, i am not sure if it's the right place here. :)

File: docs/fields/image.md [hunk 2]
```diff
@@ -35,21 +35,22 @@ field.
 
 * `extensions` Allows you to restrict users to only be able to upload files with
   certain file extensions.
-* `attrib` Can be set to either of the following:
+* `attrib` Can be set to a custom attribute, such as alt-texts, titles, captions or data-attributes. For example:
   * `alt` Show a text field for the `alt` parameter
   * `title` Show a text field for the `title` parameter
-  * `[title, alt]` Show both fields
+  * `caption` Show a custom field for a `caption` this image uses in your template
+  * `[alt, title, caption]` Add more then one attribute
```
Comment (JarJak) [inline docs/fields/image.md]: than*

File: docs/fields/image.md [hunk 3]
```diff
@@ -35,21 +35,22 @@ field.
 
 * `extensions` Allows you to restrict users to only be able to upload files with
   certain file extensions.
-* `attrib` Can be set to either of the following:
+* `attrib` Can be set to a custom attribute, such as alt-texts, titles, captions or data-attributes. For example:
   * `alt` Show a text field for the `alt` parameter
   * `title` Show a text field for the `title` parameter
-  * `[title, alt]` Show both fields
+  * `caption` Show a custom field for a `caption` this image uses in your template
```
Comment (JarJak) [inline docs/fields/image.md]: that image will use in your template*

**Review by JarJak — CHANGES_REQUESTED** [review]

File: docs/fields/image.md (hunk 2 above)
Comment (ankedsgn) [reply docs/fields/image.md]: meh. f*ck. :)

**Review by JarJak — APPROVED** [review]

**Comment by JarJak** [comment]
Thank you @anketwokings !

**Comment by ankedsgn** [comment]
@JarJak Thanks for taking the time to check both the PRs! Have a good weekend!

**Pull request merged**